service/src/main/java/bio/terra/workspace/app/configuration/external/PolicyServiceConfiguration.java

```diff
@@ -39,12 +51,27 @@ public String getClientCredentialFilePath() {
   }
 
   public String getAccessToken() throws IOException {
-    try (FileInputStream fileInputStream = new FileInputStream(clientCredentialFilePath)) {
-      GoogleCredentials credentials =
-          ServiceAccountCredentials.fromStream(fileInputStream)
-              .createScoped(POLICY_SERVICE_ACCOUNT_SCOPES);
-      AccessToken token = credentials.refreshAccessToken();
-      return token.getTokenValue();
+    try {
+      if (features.isAzureControlPlaneEnabled()) {
+        TokenCredential credential = new DefaultAzureCredentialBuilder().build();
```

Comment:
This block seems to be repeated several times in this PR, can it be extracted to a common utility method?